src/java/org/apache/cassandra/tools/nodetool/TableHistograms.java

```diff
@@ -60,21 +61,25 @@ public void execute(NodeProbe probe)
             allTables.put(entry.getKey(), entry.getValue().getTableName());
         }
 
-        if (args.size() == 2)
+        if (args.size() == 2 &&
+            args.stream().noneMatch(arg -> arg.contains(".")))
```

Comment:
I think this is cleaner without the line break, I can remove it on commit if you agree.